### PR TITLE
Add HLAfreq recipe

### PR DESCRIPTION
**Pull request by DAWells**
Add HLAfreq recipe building from pypi

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by DAWells** [comment]
@BiocondaBot please add label

File: recipes/hlafreq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - bs4
+    - matplotlib-base
+    - numpy
+    - pandas
+    - pip
+    - pymc
+    - python >=3.8
+    - requests
+    - scipy
```
Comment (bgruening) [inline recipes/hlafreq/meta.yaml:27]:
```suggestion
    - python >=3.8
    - pip
```

**Comment by DAWells** [comment]
@BiocondaBot please fetch artifacts

**Comment by DAWells** [comment]
@BiocondaBot please fetch artifacts

**Comment by DAWells** [comment]
@BiocondaBot please fetch artifacts

**Comment by DAWells** [comment]
@BiocondaBot please fetch artifacts

**Comment by DAWells** [comment]
@bioconda/python
Despite including `arviz` and `pymc >=3` in run requirements, `arviz` is not installed and `pymc` version 2.3.8 is installed. 
This means that when I install from the artifacts I cannot follow the example at https://github.com/Vaccitech/HLAfreq/blob/main/examples/paper/basic_use_case.py

**Comment by mfansler** [comment]
@DAWells how are you installing the package? Directly installing the tarball will not include dependencies. 

I get `pymc=5.5 arviz=0.15.1` when using:

```bash
mamba create -n foo -c ./packages hlafreq
```

**Comment by DAWells** [comment]
Awesome, thanks so much @mfansler! I was doing `conda create -n foo -c ./packages hlafreq` but I needed `conda create -n foo -c ./packages -c conda-forge hlafreq`.

I think this is ready to be merged then?

**Review by kbseah — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Pull request merged**